cmd/hz/app/app.go

```diff
@@ -44,6 +46,23 @@ func New(c *cli.Context) error {
 	setLogVerbose(args.Verbose)
 	logs.Debugf("args: %#v\n", args)
 
+	// determine if code needs to be regenerated
+	exists, err := util.PathExist(path.Join(args.OutDir, meta.ManifestFile))
+	if err != nil {
+		return cli.Exit(err, meta.LoadError)
+	}
+
+	if exists {
+		var isOverride bool
+		qs := overrideQuestions()
+		if err = survey.Ask(qs, &isOverride); err != nil {
```

Comment:
If it is just to achieve this function, there should be no need to introduce this library?
For instance: add a flag to enable override && refuse all new operation if there already exists a project.

Reply:
+1~Interactive methods may block the automatic execution of scripts. Perhaps if there are already existing files in the "new", we can skip them by default and print a warning log, and if the user needs to regenerate them, delete the corresponding files manually

Reply:
done